apps/web/src/pages/Bookings/Bookings.tsx

```diff
@@ -1,12 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useQuery } from '@tanstack/react-query';
 import { bookingQueryKeys } from '@/services/bookings/request';
+import FullCalendar from '@fullcalendar/react';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import styles from './Bookings.module.css';
+import { useDisclosure } from '@mantine/hooks';
+import { Drawer } from '@mantine/core';
+import { Booking } from '@/models/booking';
+
+interface event {
```

Comment:
las interfaces son con mayuscula
```suggestion
interface Event {
```